webpack/assets/javascripts/react_app/routes/common/reducerHOC/__tests__/__snapshots__/reducerHOC.test.js.snap

```diff
@@ -0,0 +1,97 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`reducerHOC should handle TEST_PAGE_DATA_FAILED 1`] = `
+Object {
+  "data": Object {
+    "hasError": false,
+    "isLoading": true,
+    "message": Object {
+      "text": "some-error",
+      "type": "error",
+    },
+  },
+  "query": Object {
+    "itemCount": 0,
+    "page": 1,
+    "perPage": 25,
+    "searchQuery": "",
+  },
+}
+`;
+
+exports[`reducerHOC should handle TEST_PAGE_DATA_RESOLVED 1`] = `
+Object {
+  "data": Object {
+    "hasError": false,
+    "isLoading": true,
```

Comment:
The problem with auto-generating snapshots is you might not notice the mistakes in the snapshots... (see below for what is missing)